packages/cli/src/commands/cluster/install/setupInboundNetworking.ts

```diff
@@ -136,38 +137,111 @@ export async function setupInboundNetworking(
               ).text(),
             }),
-            await buildDeployModuleTask({
-              taskTitle: "Deploy Ingress NGINX",
-              context,
-              env: {
-                ...process.env,
-                VAULT_ADDR: `http://127.0.0.1:${ctx.vaultProxyPort}`,
-                VAULT_TOKEN: vaultRootToken,
-              },
-              environment,
-              region,
-              module: MODULES.KUBE_INGRESS_NGINX,
-              initModule: true,
-              hclIfMissing: await Bun.file(
-                kubeNginxIngressTerragruntHcl
-              ).text(),
-              inputUpdates: {
-                ingress_domains: defineInputUpdate({
-                  schema: z.array(z.string()),
-                  update: () => [kubeDomain],
-                }),
-                sla_level: defineInputUpdate({
-                  schema: z.number(),
-                  update: () => 1,
-                }),
-              },
-              postDeployInputUpdates: {
-                sla_level: defineInputUpdate({
-                  schema: z.number().optional(),
-                  update: () => undefined,
-                })
+            {
+              task: async (ctx, task) => {
+                return task.newListr<Context>([
+                  await buildDeployModuleTask({
+                    taskTitle: "Deploy Ingress NGINX",
+                    context,
+                    env: {
+                      ...process.env,
+                      VAULT_ADDR: `http://127.0.0.1:${ctx.vaultProxyPort}`,
+                      VAULT_TOKEN: vaultRootToken,
+                    },
+                    environment,
+                    region,
+                    module: MODULES.KUBE_INGRESS_NGINX,
+                    initModule: true,
+                    hclIfMissing: await Bun.file(
+                      kubeNginxIngressTerragruntHcl
+                    ).text(),
+                    inputUpdates: {
+                      ingress_domains: defineInputUpdate({
+                        schema: z.array(z.string()),
+                        update: () => [kubeDomain],
+                      }),
+                      sla_level: defineInputUpdate({
+                        schema: z.number(),
+                        update: () => 1,
+                      }),
+                    },
+                    postDeployInputUpdates: {
+                      sla_level: defineInputUpdate({
+                        schema: z.number().optional(),
+                        update: () => undefined,
+                      })
+                    }
+                  }),
+                  {
+                    title: "Resetting Cert Manager",
+                    task: async () => {
+                      let attempts = 0;
+                      const maxAttempts = 10;
+                      const retryDelay = 90000;
+
+                      const kc = new k8s.KubeConfig();
+                      kc.loadFromDefault();
```

Comment:
where are you setting the context?

Reply:
I was curious about this as well lol. The [docs](https://kubernetes-client.github.io/javascript/) aren't helpful in this case. I'll see if I can find examples.